lib/task/import/csvAuthorityRecordImportTask.class.php

```diff
@@ -65,6 +65,37 @@ public function execute($arguments = [], $options = [])
             QubitTaxonomy::DESCRIPTION_DETAIL_LEVEL_ID => 'detailLevelTypes',
         ]);
 
+        // Define helper function used in post-save logic
+        $relateTermFunction = function ($import, $column, $taxonomyId) {
+            if (!empty($import->rowStatusVars[$column])) {
+                $places = explode('|', $import->rowStatusVars[$column]);
```

Comment:
Heh, I think `$terms` (or similar) is preferable to `$places` now that this is a generic method. :)

Reply:
D'oh! Good point. ;)